EXAMPLES/moving_sources_acoustic/DATA/Par_file

```diff
@@ -0,0 +1,389 @@
+#-----------------------------------------------------------------------------
```

Comment:
great idea to add an example! will use this one also for continuous testing in travis/jenkins

Reply:
Cool! Thank you! You can add it in GPU and CPU, both should work fine (even with several GPUs which was a little bit tricky because the source can go from one slice to another)